plugins/Terminal/js/components/seedprompt.js

```diff
@@ -22,6 +22,8 @@ export default class WalletSeedPrompt extends React.Component {
 					'dictionary': 'english',
 				}))
 				siac.end()
+				e.target.value = ''
```

Comment:
Is `e.target.value =''` necessary?  The input's value isn't stored as state, so it should be cleared every time the seed prompt is reopened.